EventTicketingSystem.DataAccess/Services/VenueRepository.cs

```diff
@@ -0,0 +1,61 @@
+﻿using EventTicketingSystem.DataAccess.Interfaces;
+using EventTicketingSystem.DataAccess.Models.Context;
+using EventTicketingSystem.DataAccess.Models.Entities;
+using Microsoft.EntityFrameworkCore;
+
+namespace EventTicketingSystem.DataAccess.Services
+{
+    internal class VenueRepository : IVenueRepository
+    {
+        private readonly DatabaseContext _context;
+
+        public VenueRepository(DatabaseContext context)
+        {
+            _context = context;
+        }
+
+        public async Task<Venue> Find(int id)
+        {
+            return await _context.Venues.FindAsync(id);
```

Comment:
Find looks like DRY violation. To base class please

Reply:
Done